indra/llcommon/threadpool.cpp

```diff
@@ -39,6 +39,11 @@ void LL::ThreadPool::start()
                 run(tname);
             });
     }
+
+    // Special workflow for LLWindowWin32Thread - it's close() should be called explicitly
+    if (mExplicitShutdown)
+        return;
```

Comment:
What issues close() call in such case?

Reply:
> What issues close() call in such case?
As iit was said in the comment above, the destructor of the class ThreadPool calls this method in any case:
```
LL::ThreadPool::~ThreadPool()
{
    close();
}
```